src/libYARP_manager/include/yarp/manager/executable.h

```diff
@@ -89,7 +96,12 @@ class Executable : public BrokerEventSink
     ResourceContainer& getResources(void) { return resources; }
 
     RSTATE state(void);
+    BrokerType getBrokerType();
+    bool shouldChangeBroker();
     Broker* getBroker(void) { return broker; }
+    void setAndInitializeBroker(Broker* _broker);
```

Comment:
`setAndInitializeBroker` -> `updatreBroker` ?

Reply:
I chose this name to make explicit the fact that the broker is set and initialized :thinking: